mars/dataframe/arithmetic/add.py

```diff
@@ -75,16 +77,14 @@ def _set_inputs(self, inputs):
 
 
 def add(df, other, axis='columns', level=None, fill_value=None):
-    if isinstance(other, (DATAFRAME_TYPE, SERIES_TYPE)) or np.isscalar(other):
-        op = DataFrameAdd(axis=axis, level=level, fill_value=fill_value, lhs=df, rhs=other)
-    else:
-        raise NotImplementedError('Only support add with dataframe, series or scalar!')
+    if isinstance(other, (list, tuple, np.ndarray, TENSOR_TYPE)):
+        other = Series(other)
```

Comment:
`intializer.Series` seems a shell that wraps everything, how about put the import here to prevent potential loop import.

Reply:
Fixed.